DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -13,9 +15,20 @@ DmlDispatchable::DmlDispatchable(
     const Model::DmlDispatchableDesc& desc,
     const Dispatchable::Bindings& initBindings,
     IDxDispatchLogger* logger
-    ) : m_name(name), m_device(device), m_desc(desc), m_initBindings(std::move(initBindings)), m_logger(logger)
+    ) : m_name(name), m_device(device), m_desc(desc), 
+        m_initBindings(std::move(initBindings)), m_logger(logger), m_isSerializedGraph(false)
+{
+    m_bindPoints = desc.bindPoints;
```

Comment:
Can we initialize `m_bindPoints` same as other class level variables like `m_name`?